README.md

```diff
@@ -103,17 +103,22 @@ kubectl apply -f ./examples/kubernetes/demo-pod.yaml
 Setup GCP service account first and setup Kubernetes cluster
 ```
 $ ./deploy/project_setup.sh
-$ ./deploy/kubernetes/cluster_setup.sh
-```
-
-### Manual
-```
 # Build the Filestore CSI driver image and push to a container registry.
 $ PROJECT=your-gcp-project make build-image-and-push
-# Modify manifests under deploy/kubernetes/manifests to use development image
-$ ./deploy/kubernetes/driver_start.sh
+
+# The base manifests are listed under deploy/kubernetes/base.
+# The overlays (e.g prow-gke-release-staging-head, prow-gke-release-staging-rc, stable, noauth) are listed under deploy/kubernetes/overlays
```

Comment:
Let's use dev rather than noauth. I think we want to discourage downloading SA keys generally, so I'm ok with changing from the convention of the PD CSI driver and making the default dev overlay use scopes instead.

Reply:
Done